koii.py

```diff
@@ -48,17 +49,17 @@ def _format(self, paths: List[Path]) -> None:
         path_header_fill = " " * (self._TAB_SIZE - len("PATH"))
 
         print(" ")
-        method_header = f"\033[1;35;40mMETHOD {method_header_fill}"
-        path_header = f"\033[1;35;40mPATH {path_header_fill}"
+        method_header = f"{Fore.LIGHTMAGENTA_EX} METHOD {method_header_fill}"
```

Comment:
I think we need to remove the space between `Fore.LIGHTMAGENTA_EX}` and `METHOD`.

Reply:
Or if we add the space, maybe it should be consistent everywhere. SO we decide if we are adding the space or not and make it consistent.

Reply:
cc @zachang